### PR TITLE
BackupV3_Octopus

### PR DESCRIPTION
**Pull request by cristinamanum**
Fixed issues found  during deployment.

### CODE REVIEW

File: src/CopyAzureContainer/Scripts/backupv3storage.cmd [hunk 1]
```diff
@@ -1,11 +1,11 @@
 @echo OFF
 
-cd Ng
+<<<<<<< HEAD
```
Comment (shishirx34) [inline src/CopyAzureContainer/Scripts/backupv3storage.cmd]: merge conflict?

File: src/CopyAzureContainer/CopyAzureContainer.nuspec [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    <version>$version$</version>
+    <title>CopyAzureContainer</title>
+    <authors>.NET Foundation</authors>
+    <owners>.NET Foundation</owners>
+    <description>CopyAzureContainer</description>
+    <copyright>Copyright .NET Foundation</copyright>
+  </metadata>
+  <files>
+    <file src="bin\$configuration$\*.*" target="bin"/>
+
```
Comment (shishirx34) [inline src/CopyAzureContainer/CopyAzureContainer.nuspec]: nit remove empty line?

File: src/CopyAzureContainer/Scripts/backupv3storage.cmd (hunk 1 above)
Comment (cristinamanum) [inline src/CopyAzureContainer/Scripts/backupv3storage.cmd]: Fixed

File: src/CopyAzureContainer/CopyAzureContainer.nuspec (hunk 2 above)
Comment (cristinamanum) [inline src/CopyAzureContainer/CopyAzureContainer.nuspec]: Fixed

**Pull request merged**